src/main/java/com/epam/prejap/teatrees/game/Printer.java

```diff
@@ -42,4 +42,19 @@ void endRow() {
     void border(int width) {
         out.println("+" + "-".repeat(width) + "+");
     }
+
+    public void printWelcomeMsg() {
+        out.println("""
+                _|          _|              _|                                                   \s
+                _|          _|     _|_|     _|     _|_|_|     _|_|     _|_|_|  _|_|       _|_|   \s
+                _|    _|    _|   _|_|_|_|   _|   _|         _|    _|   _|    _|    _|   _|_|_|_| \s
+                  _|  _|  _|     _|         _|   _|         _|    _|   _|    _|    _|   _|       \s
+                    _|  _|         _|_|_|   _|     _|_|_|     _|_|     _|    _|    _|     _|_|_| \s
+                """);
+        try {
+            Thread.sleep(3000);
+        } catch (InterruptedException e) {
+            System.err.println(e.getMessage());
```

Comment:
InterruptedException is thrown when a thread is interrupted by another thread while waiting for some event to happen (such as timeout in this case). What is the purpose of printing the message from the exception, if, according to the issue description, sleeping is meant to be interrupted?

Reply:
done